rust/processor/src/schema.rs

```diff
@@ -268,6 +270,25 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    current_fungible_asset_balances (storage_id) {
+        #[max_length = 66]
+        storage_id -> Varchar,
```

Comment:
Is this auto-generated or does it have anything to do with the account address in case of Coin vs the FungibleStore object in case of FungibleAsset?

Reply:
For FA: it's the MetadataStore address
For V1: it's the hash of owner and coin type (e.g. 0xbowen::0x1::aptos_coin::AptosCoin)

Reply:
For FA: that's not necessarily true if it's a hash of owner addr + type like you mentioned in an earlier comment. That's only true for primary fungible stores. Secondary store addresses are arbitrary

Reply:
Yea but it's the same pattern.

Reply:
Basically in v1 there is only primary stores.